cmd/crc/cmd/version.go

```diff
@@ -21,5 +44,15 @@ var versionCmd = &cobra.Command{
 }
 
 func runPrintVersion(arguments []string) {
-	fmt.Printf("version: %s+%s\n", crcPkg.GetCRCVersion(), crcPkg.GetCommitSha())
+	output := fmt.Sprintf("%s+%s\n", crcPkg.GetCRCVersion(), crcPkg.GetCommitSha())
+	if outputFormat == "json" {
+		strOutput := outputPkg.VersionOutputStruct{output}
```

Comment:
the output struct is not needed inside the `output` package. the command in this case can handle. however, the `output` library can be instructed what needs to be printed in a normal case FROM this struct if needed.